### DISCUSSION TITLE
Enquire about the implications of setting a parameter Days Transactions Exisit in POS functionality Profile.

### DISCUSSION THREAD

**Discussion by SamuelH-MCS** [microsoft/Dynamics365Commerce.Solutions#345] [Q&A]
Enquire about the implications of setting a parameter value to 5 days while their POS machine had been offline for weeks due to network issues. They wanted to know if the headquarters would still receive all offline transactions from the past few weeks, as well as the records of blind closed shifts.

**Comment by Gopikrish25** [comment]
I believe that offline transactions that have not yet been synced to the channel database and headquarters will not be affected by this parameter. This parameter only impacts the records that exist in the Channel DB up to a specific point in time. I will wait for feedback from others to see if I am mistaken in my assumption.

I recommend setting the parameter based on the store's return policy, such as 30 days or 90 days.
